Sources/ProjectDescription/ProjectOptions.swift

```diff
@@ -12,8 +12,8 @@ extension Project {
         /// Configures the development region.
         public var developmentRegion: String?
 
-        /// Disables generating Bundle accessors.
-        public var disableBundleAccessors: Bool
+        /// Defines if and how bundle accessors are generated
+        public var bundleAccessorsOptions: BundleAccessorOptions
```

Comment:
This is a public API breaking change - while I think its ok, I can keep support for the boolean flag.
I was mainly thinking it is checked internally and set externally and as such didn't see an issue removing it

Reply:
Let's keep the old one, mark it as deprecated to tell users to use `bundleAccessorOptions`, and default to `bundleAccessorOptions` with a fallback on `disableBundleAccessors`. We've recently gone through a handful of breaking changes with Tuist 4 and we should give users some breaking changes calm.